crates/extensions/tedge_log_manager/src/config.rs

```diff
@@ -10,6 +10,7 @@ use tedge_config::tedge_toml::ReadError;
 use tedge_mqtt_ext::Topic;
 use tedge_mqtt_ext::TopicFilter;
 
+pub const DEFAULT_PLUGIN_DIR: &str = "log-plugins";
 pub const DEFAULT_PLUGIN_CONFIG_FILE_NAME: &str = "tedge-log-plugin.toml";
 pub const DEFAULT_PLUGIN_CONFIG_DIR_NAME: &str = "plugins/";
```

Comment:
What's the difference between `log-plugins` and `plugins`?

Reply:
That terminology is legacy, from a time when log management was part of a standalone `tedge-log-plugin`. Once it was merged with `tedge-agent`, the term `plugin` was not relevant anymore. When the existing file-based log management functionality is moved to a `file` plugin, then this `DEFAULT_PLUGIN_CONFIG_FILE_NAME` would become `DEFAULT_FILE_PLUGIN_CONFIG_FILE_NAME`.